src/SprykerEco/Zed/Computop/ComputopConfig.php

```diff
@@ -256,32 +261,58 @@ public function isRefundShipmentPriceEnabled()
     /**
      * @api
      *
-     * @return array
+     * @return array<string>
      */
-    public function getBeforeCaptureStatuses()
+    public function getBeforeInitStatuses(): array
     {
         return [
             $this->getOmsStatusNew(),
-            $this->getOmsStatusAuthorized(),
-            $this->getOmsStatusAuthorizationFailed(),
-            $this->getOmsStatusCancelled(),
         ];
     }
 
     /**
      * @api
      *
-     * @return array
+     * @return array<string>
      */
-    public function getBeforeRefundStatuses()
+    public function getBeforeAuthorizeStatuses(): array
     {
         return [
             $this->getOmsStatusNew(),
-            $this->getOmsStatusAuthorized(),
-            $this->getOmsStatusCaptured(),
+            $this->getOmsStatusInitialized(),
+            $this->getAuthorizeRequestOmsStatus(),
+            $this->getOmsStatusAuthorizationFailed(),
         ];
     }
 
+    /**
+     * @api
+     *
+     * @return array<string>
+     */
+    public function getBeforeCaptureStatuses(): array
```

Comment:
adding return type to an existing config method is a BC-break
please revert it

Reply:
Fixed